pumps/csv.go

```diff
@@ -126,3 +128,11 @@ func (c *CSVPump) WriteData(data []interface{}) error {
 	writer.Flush()
 	return nil
 }
+
+func (c *CSVPump) SetTimeout(timeout int) {
+	c.timeout = timeout
+}
+
+func (c *CSVPump) GetTimeout() int {
+	return c.timeout
+}
```

Comment:
I don't see anywhere where they are used, what is the point of having this?

Reply:
needed for the new Pump interface. It's used in initializePumps / writeToPumps to set / get the timeout of the pumps.